src/components/Pomodoro.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/Pomodoro.css";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import ReactPlayer from "react-player";
```

Comment:
Why are these exceptions being included? Taking them out results in no errors, so they should be removed